src/components/profile/profile.js

```diff
@@ -64,15 +80,125 @@ class Profile extends Component {
         // } button not needed yet
     }
 
+    addTILForm() {
+        if (this.state.loggedInUserProfile) {
+            return <div className="til_form">
+            <form className="til_form_enter" onSubmit={this.sendNewTIL} >
+                <Header title="Things I Learned:" className="til_header"/>
+                <ul className="form_wrapper">
+                    <li className="form_row">
+                        <TextareaInput className="TIL_textarea" rows="3" name="New TIL"
+                                       placeholder="What did you learn today?" handleChange={this.handleChange}/>
+                    </li>
+                </ul>
+                {this.databaseErrorMessage()}
+                <p>Max 255 characters</p>
+                <Button name="Add TIL"/>
+            </form>
+            </div>
+        } else {
+            return <h1>Placeholder for follow</h1>
+        }
+    }
+
+    getTilPosts = async () => {
+        return axios({
+            method: "GET",
+            withCredentials: true,
+            url: "http://localhost:3001/til/" + this.state.url
+        }).then( response => {
+            if(response.data.success){
+                this.setState({userTils: response.data.info})
+            } else {
+                this.setState({userTils: 'Nothing to see here'})
+            }
+        }).catch( err => {
+            this.setState({databaseError: 'Sorry your TILs were not retrieved.'})
+        })
+    }
+
+    formatDate(date) {
+       const splitTimestamp = date.split(/[T.]/);
+       const dateOfTil = splitTimestamp[0];
+       let timeOfTil = splitTimestamp[1];
+       timeOfTil = timeOfTil.substring(0, timeOfTil.length-3)
+       return dateOfTil + ' at ' + timeOfTil
+    }
+
+
+    displayTILs() {
+        const userTils = this.state.userTils;
+        userTils.reverse()
+       return (
```

Comment:
woah this uneven tabbing makes me feel nauseous

Reply:
ha. ha. ha